### PR TITLE
Show most recent scheduling errors

### PR DESCRIPTION
**Pull request by satishwaghela**
On the TaskName->TableName page, the SCHEDULING ERRORS was previously TBD. Now we have the API for it: GET /tasks/generator/{tableNameWithType}/{taskType}/debug

### CODE REVIEW

**Comment by npawar** [comment]
Just tried this out. I'm not seeing the exception message be picked up. For example, here the response has only errors, and no success TS, but Scheduling Errors remains blank 
Screen Shot 2022-08-04 at 10 48 16 PM


A sample response will look like this, for your reference:
```
[
  {
    "tableNameWithType": "githubEvents_OFFLINE",
    "taskType": "MergeRollupTask",
    "mostRecentSuccessRunTS": [],
    "mostRecentErrorRunMessages": {
      "2022-08-05T05:44:00.267Z": "java.lang.IllegalStateException: Dummy exception\n\tat org.apache.pinot.plugin.minion.tasks.mergerollup.MergeRollupTaskGenerator.generateTasks(MergeRollupTaskGenerator.java:135)\n\tat org.apache.pinot.controller.helix.core.minion.PinotTaskManager.scheduleTask(PinotTaskManager.java:536)\n\tat org.apache.pinot.controller.helix.core.minion.PinotTaskManager.scheduleTask(PinotTaskManager.java:620)\n\tat org.apache.pinot.controller.helix.core.minion.CronJobScheduleJob.execute(CronJobScheduleJob.java:53)\n\tat org.quartz.core.JobRunShell.run(JobRunShell.java:202)\n\tat org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)\n",
      "2022-08-05T05:45:00.005Z": "java.lang.IllegalStateException: Dummy exception\n\tat org.apache.pinot.plugin.minion.tasks.mergerollup.MergeRollupTaskGenerator.generateTasks(MergeRollupTaskGenerator.java:135)\n\tat org.apache.pinot.controller.helix.core.minion.PinotTaskManager.scheduleTask(PinotTaskManager.java:536)\n\tat org.apache.pinot.controller.helix.core.minion.PinotTaskManager.scheduleTask(PinotTaskManager.java:620)\n\tat org.apache.pinot.controller.helix.core.minion.CronJobScheduleJob.execute(CronJobScheduleJob.java:53)\n\tat org.quartz.core.JobRunShell.run(JobRunShell.java:202)\n\tat org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573)\n"
    }
  }
]
```

**Comment by npawar** [comment]
cc @saurabhd336

**Comment by npawar** [comment]
@joshigaurava please help review

File: pinot-controller/src/main/resources/app/pages/TaskDetail.tsx
Comment (joshigaurava) [inline pinot-controller/src/main/resources/app/pages/TaskDetail.tsx:107]: Please remove the code if it's not going to be used rather than commenting it out.

**Comment by joshigaurava** [comment]
Please eliminate code that is no longer required rather than simply commenting it out.

**Comment by npawar** [comment]
Looks like the text is spilling out of the Scheduling errors box. Can you just make the text box take up all the space from from left to right, instead of right aligned?
Also, the timezone of the Previous Fire Time and Next Fire Time is incorrect. The response says it is in America/Los_Angeles, but UI shows as GMT
Screen Shot 2022-08-05 at 2 12 24 PM

**Comment by satishwaghela** [comment]
> Also, the timezone of the Previous Fire Time and Next Fire Time is incorrect. The response says it is in America/Los_Angeles, but UI shows as GMT

@npawar, Server is sending PreviousFireTime and NextFireTime in epoch format that's why creating date object from that UI shows user's local time.

**Review by npawar — APPROVED** [review]
lgtm functionally

**Comment by npawar** [comment]
> lgtm functionally

Will merge once @joshigaurava approves

File: pinot-controller/src/main/resources/app/pages/TaskQueueTable.tsx
Comment (joshigaurava) [inline pinot-controller/src/main/resources/app/pages/TaskQueueTable.tsx:205]: @satishwaghela please don't break the words here. Introduce horizontal scroll bars. Breaking the class path will make it unreadable.

**Review by joshigaurava — CHANGES_REQUESTED** [review]
My previous comment was to remove unused code. It's just that I didn't point all the places where we need to remove unused code. Please go through the entire PR and remove the code instead of commenting it out.

**Comment by joshigaurava** [comment]
@satishwaghela can you please post an updated screenshot? For a UI change, it helps in correlating code with the output.

**Comment by satishwaghela** [comment]
> @satishwaghela can you please post an updated screenshot? For a UI change, it helps in correlating code with the output.

image

**Review by joshigaurava — APPROVED** [review]

**Pull request merged**
